### ISSUE TITLE
console.log not displaying in bash terminal

### ISSUE THREAD

**Issue by kenlnetherland** [microsoft/vscode#212840]
This may work for Powershell, not for Bash:

              @oliversalzburg if you do not want to switch to the "legacy" protocol, you can use a different console and continue to use the "inspector" protocol: just add a `"console": "integratedTerminal"` to your launch config.

_Originally posted by @weinand in https://github.com/microsoft/vscode/issues/30842#issuecomment-323787963_

**Comment by meganrogge** [comment]
adding @roblourens as he has context here. not sure what the problem is

**Comment by kenlnetherland** [comment]
I have an Angular web front-end.  I use VS Code and the Bash terminal, since I have to support Linux.  Javascript methods  console.log and console.debug has no effect (output) on the integrated Bash terminal in either Output, Debug Console, or Terminal windows.

**Comment by meganrogge** [comment]
thanks for that info. Can you pls provide simple steps to reproduce the problem and state your expectation vs actual?

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by FatNerdPeng** [comment]
Hello,have you solved this problem? I have also encountered this issue. 
I am currently using VS code ```1.90.0``` version. In simple JavaScript code, ```console. log()``` can output normally on the terminal, but in large projects I built with ```Nestjs```, the terminal will not output the content of ```console. log```. But I haven't changed my ```Nestjs``` version, nor have I changed any code. I just haven't changed the code for a while, and then updated the VS code version. However, during debugging, I found that the terminal no longer outputs ```console. log()``` content. I tried to lower the VS code version but it didn't work.
